config.py

```diff
@@ -0,0 +1,14 @@
+import secrets
+
+class Config:
```

Comment:
use pydantic basesettings - in the future not right now you didnt study pydantic yet

Reply:
fixed

Reply:
DEBUG = False
    TESTING = False
    VERSION = "0.1.3"
    
    do you have a debug environment requiring a different config? already commented on this.. think before copying
    
    again are you version 0.1.3? you dont even have versions it was an example code not to copy..
    
    think before you copy clean your code to be only whats nessacary

Reply:
done, i thing i did right